internal/params/config.go

```diff
@@ -47,7 +47,7 @@ var (
 		QuickUnlockEpoch:           big.NewInt(191),
 		FiveSecondsEpoch:           big.NewInt(230),
 		TwoSecondsEpoch:            big.NewInt(366), // Around Tuesday Dec 8th 2020, 8AM PST
-		SixtyPercentEpoch:          EpochTBD,
+		SixtyPercentEpoch:          big.NewInt(530), // Around Monday Apr 12th 2021, 22:30 UTC
 		RedelegationEpoch:          big.NewInt(290),
 		NoEarlyUnlockEpoch:         EpochTBD,
```

Comment:
please also turn on NoEarlyUnlockEpoch = big.NewInt(530)

Reply:
got it.